普通に端末で実行はできるんですが:app:assembleDebugが動いてなかったので見ています 👀 

```
> Task :app:mergeExtDexDebug FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeExtDexDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform artifact 'core.aar (androidx.core:core:1.2.0-alpha02)' to match attributes {artifactType=android-dex, dexing-enable-desugaring=false, dexing-is-debuggable=true, dexing-min-sdk=21}.
      > Execution failed for DexingNoClasspathTransform: /Users/takahirom/.gradle/caches/transforms-2/files-2.1/e65687af5a207708d38aa674f6fb7b59/jetified-classes.jar.
         > Error while dexing.
   > Failed to transform artifact 'core.aar (androidx.core:core:1.2.0-alpha02)' to match attributes {artifactType=android-dex, dexing-enable-desugaring=false, dexing-is-debuggable=true, dexing-min-sdk=21}.
      > Execution failed for DexingNoClasspathTransform: /Users/takahirom/.gradle/caches/transforms-2/files-2.1/e65687af5a207708d38aa674f6fb7b59/jetified-classes.jar.
```

--- a/Raichou/app/build.gradle
+++ b/Raichou/app/build.gradle
@@ -34,26 +34,26 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.0.2'
-    implementation 'androidx.core:core-ktx:1.0.1'
+    implementation 'androidx.core:core-ktx:1.1.0-rc02'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.material:material:1.0.0'
-    implementation 'androidx.recyclerview:recyclerview:1.1.0-alpha04'
+    implementation 'androidx.recyclerview:recyclerview:1.1.0-beta01'
     implementation 'com.squareup.picasso:picasso:2.71828'
-    implementation 'com.google.firebase:firebase-core:16.0.8'
-    implementation 'com.google.firebase:firebase-storage:16.1.0'
-    implementation 'com.google.firebase:firebase-firestore:18.2.0'
+    implementation 'com.google.firebase:firebase-core:17.0.0'
+    implementation 'com.google.firebase:firebase-storage:18.0.0'
+    implementation 'com.google.firebase:firebase-firestore:20.1.0'
     implementation 'com.jakewharton.timber:timber:4.7.1'
-    implementation "androidx.navigation:navigation-fragment:2.1.0-alpha05"
-    implementation "androidx.navigation:navigation-ui:2.1.0-alpha05"
-    implementation "androidx.navigation:navigation-fragment-ktx:2.1.0-alpha05"
-    implementation "androidx.navigation:navigation-ui-ktx:2.1.0-alpha05"
+    implementation "androidx.navigation:navigation-fragment:2.1.0-alpha06"
+    implementation "androidx.navigation:navigation-ui:2.1.0-alpha06"
+    implementation "androidx.navigation:navigation-fragment-ktx:2.1.0-alpha06"
+    implementation "androidx.navigation:navigation-ui-ktx:2.1.0-alpha06"
 
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.2.1'
 
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 }
 apply plugin: 'com.google.gms.google-services'

--- a/Raichou/build.gradle
+++ b/Raichou/build.gradle
@@ -1,17 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.21'
+    ext.kotlin_version = '1.3.31'
     repositories {
         google()
         jcenter()
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0-beta02'
+        classpath 'com.android.tools.build:gradle:3.5.0-beta05'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.2.0'
-        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:2.1.0-alpha05"
+        classpath 'com.google.gms:google-services:4.3.0'
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:2.1.0-alpha06"
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }

